contracts/UMA/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -216,11 +216,11 @@ contract PricelessPositionManager is Testable, Lockable {
         bytes32 _priceIdentifier,
         FixedPoint.Unsigned memory _minSponsorTokens,
         FixedPoint.Unsigned memory _ooReward,
-        address _timerAddress,
         address _financialProductLibraryAddress,
-        bytes memory _ancillaryData
+        bytes memory _ancillaryData,
+        address _owner
     )
-        Testable(_timerAddress)
+        Testable(address(0))
```

Comment:
@RojhatToptamus I think since we are now touching this part of code, let's remove Testable contract all together. We are not using it anyways.